### ISSUE TITLE
About classification_loss?

### ISSUE THREAD

**Issue by hongzhidu** [lvwj19/PPR-Net-plus#7]
Hi, 
Thanks for your excellent work!

May I consult you about why use cross entropy as the loss of semantic segmentation? If the label of each object is not fixed during training, will this loss function be invalid？

**Comment by lvwj19** [comment]
In our experiments, the semantic id for each object was fixed during training, i.e., the objects of the same class had the same semantic id. I am curious, in which case that the semantic label of the object would be not fixed during training.

**Comment by hongzhidu** [comment]
Thanks for your reply!  I just think the semantic id is unnecessary in this task.  It might be more appropriate to do instance segmentation.

**Comment by lvwj19** [comment]
In fact, semantic labels are common in these tasks, such as semantic segmentation for background and foreground point clouds in the stacked scene.

**Comment by hongzhidu** [comment]
Thanks again for your reply!
